app/views/categories/show.html.erb

```diff
@@ -1,23 +1,25 @@
 <h1>Products in <%=@category.name%> Category</h1>
 
 <% @category.products.each do |product| %>
-  <ul>
-    <li>
-      <%=image_tag(product.photo_url)%>    
-    </li>
-    <li>
-      <h3><%= link_to product.name, product_path(product.id) %></h3>
-    </li>
-    <li>
-      <%= product.description%>
-    </li>
-    <li>
-      <%= number_to_currency(product.price)%>
-    </li>
-    <li>
-      <%= product.quantity%>
-    </li>
-  </ul>
+  <% if product.retired == false%>
+    <ul>
+      <li>
+        <%=image_tag(product.photo_url)%>    
```

Comment:
Probably should add an alt (I haven't been doing that either but we should get in the habit for accessibility)

Reply:
Good idea! Amy's fixed this, she says